src/ProblemDetailsResponseFactory.php

```diff
@@ -79,36 +82,36 @@ class ProblemDetailsResponseFactory
-        StatusCode::STATUS_PRECONDITION_REQUIRED              => 'Precondition Required',
-        StatusCode::STATUS_TOO_MANY_REQUESTS                  => 'Too Many Requests',
-        StatusCode::STATUS_REQUEST_HEADER_FIELDS_TOO_LARGE    => 'Request Header Fields Too Large',
-        444                                                   => 'Connection Closed Without Response',
-        StatusCode::STATUS_UNAVAILABLE_FOR_LEGAL_REASONS      => 'Unavailable For Legal Reasons',
-        499                                                   => 'Client Closed Request',
+        StatusCode::STATUS_BAD_REQUEST                     => 'Bad Request',
+        StatusCode::STATUS_UNAUTHORIZED                    => 'Unauthorized',
+        StatusCode::STATUS_PAYMENT_REQUIRED                => 'Payment Required',
+        StatusCode::STATUS_FORBIDDEN                       => 'Forbidden',
+        StatusCode::STATUS_NOT_FOUND                       => 'Not Found',
+        StatusCode::STATUS_METHOD_NOT_ALLOWED              => 'Method Not Allowed',
+        StatusCode::STATUS_NOT_ACCEPTABLE                  => 'Not Acceptable',
+        StatusCode::STATUS_PROXY_AUTHENTICATION_REQUIRED   => 'Proxy Authentication Required',
+        StatusCode::STATUS_REQUEST_TIMEOUT                 => 'Request Timeout',
+        StatusCode::STATUS_CONFLICT                        => 'Conflict',
+        StatusCode::STATUS_GONE                            => 'Gone',
+        StatusCode::STATUS_LENGTH_REQUIRED                 => 'Length Required',
+        StatusCode::STATUS_PRECONDITION_FAILED             => 'Precondition Failed',
+        StatusCode::STATUS_PAYLOAD_TOO_LARGE               => 'Payload Too Large',
+        StatusCode::STATUS_URI_TOO_LONG                    => 'Request-URI Too Long',
+        StatusCode::STATUS_UNSUPPORTED_MEDIA_TYPE          => 'Unsupported Media Type',
+        StatusCode::STATUS_RANGE_NOT_SATISFIABLE           => 'Requested Range Not Satisfiable',
+        StatusCode::STATUS_EXPECTATION_FAILED              => 'Expectation Failed',
+        StatusCode::STATUS_IM_A_TEAPOT                     => 'I\'m a teapot',
+        StatusCode::STATUS_MISDIRECTED_REQUEST             => 'Misdirected Request',
+        StatusCode::STATUS_UNPROCESSABLE_ENTITY            => 'Unprocessable Entity',
+        StatusCode::STATUS_LOCKED                          => 'Locked',
+        StatusCode::STATUS_FAILED_DEPENDENCY               => 'Failed Dependency',
+        StatusCode::STATUS_UPGRADE_REQUIRED                => 'Upgrade Required',
+        StatusCode::STATUS_PRECONDITION_REQUIRED           => 'Precondition Required',
+        StatusCode::STATUS_TOO_MANY_REQUESTS               => 'Too Many Requests',
+        StatusCode::STATUS_REQUEST_HEADER_FIELDS_TOO_LARGE => 'Request Header Fields Too Large',
+        444                                                => 'Connection Closed Without Response',
```

Comment:
Do these have static declarations now, perchance?

Reply:
They don't seem to be defined in the `StatusCodeInterface`.